packages/core/src/lib/implementation/execute-plugin.unit-test.ts

```diff
@@ -176,18 +186,18 @@ describe('executePlugins', () => {
             outputTransform: (outputs: unknown): Promise<AuditOutputs> =>
               Promise.resolve([
                 {
-                  slug: (outputs as AuditOutputs)[0]!.slug,
-                  score: 0.3,
-                  value: 16,
-                  displayValue: '16.0.0',
+                  slug: (outputs as AuditOutputs)[0]?.slug || '',
```

Comment:
No conditionals in tests. This is probably another consequence of ESLint thinking this is production code.
```suggestion
                  slug: (outputs as AuditOutputs)[0]!.slug,
```